src/plugins/reporter-rhtsupport.c

```diff
@@ -591,24 +618,32 @@ int main(int argc, char **argv)
         {
             /* -t: extract URL where we previously reported it */
             report_result_t *reported_to = get_reported_to(dump_dir_name);
-            if (!reported_to || !reported_to->url)
-                error_msg_and_die("Can't attach: problem data in '%s' "
-                        "was not reported to RHTSupport and therefore has no URL",
-                        dump_dir_name);
-            //log_warning("URL:'%s'", reported_to->url);
-            //log_warning("MSG:'%s'", reported_to->msg);
-            free(url);
-            url = reported_to->url;
-            reported_to->url = NULL;
-            free_report_result(reported_to);
+            if (reported_to && reported_to->url)
+            {
+                free(url);
+                url = reported_to->url;
+                reported_to->url = NULL;
+                free_report_result(reported_to);
+                char *msg = xasprintf(
+                    _("We found similar Red Hat support case %s. "
```

Comment:
I am not sure but I feel there should be an article  - `a similar`. Do you feel the same?